File: features/discovery/home/src/main/java/com/tokopedia/home/beranda/domain/gql/feed/GetHomeRecommendationContent.java

package com.tokopedia.home.beranda.domain.gql.feed;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.util.List;

public class GetHomeRecommendationContent {

    @SerializedName("recommendation_product")
    @Expose
    private RecommendationProduct recommendationProduct;

    public RecommendationProduct getRecommendationProduct() {
        return recommendationProduct;
    }
}

File: features/discovery/home/src/main/java/com/tokopedia/home/beranda/domain/gql/feed/GetHomeRecommendationContent.kt
package com.tokopedia.home.beranda.domain.gql.feed

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

data class GetHomeRecommendationContent(
    @SerializedName("recommendation_product", alternate = ["getHomeRecommendationProductV2"])
    @Expose
    val recommendationProduct: RecommendationProduct = RecommendationProduct()
)
